src/components/OrgView/OrgView.js

```diff
@@ -128,132 +231,192 @@ function OrgView({
+                  height: calc(100vh - ${8 * GU}px);
+                `
+                  : ''}
               `}
             >
-              <Transition
-                config={springs.lazy}
-                items={showAppOverlay}
-                immediate={showAppOverlay}
-                enter={{ opacity: 1 }}
-                leave={{ opacity: 0 }}
-              >
-                {show =>
-                  show &&
-                  (({ opacity }) => (
-                    <AnimatedDiv
-                      style={{ opacity }}
-                      css={`
-                        position: absolute;
-                        z-index: 2;
-                        top: 0;
-                        left: 0;
-                        right: 0;
-                        bottom: 0;
-                        background: ${theme.background};
-                      `}
-                    />
-                  ))
-                }
-              </Transition>
+              <MenuPanel
+                activeInstanceId={activeInstanceId}
+                appInstanceGroups={appInstanceGroups}
+                appsStatus={appsStatus}
+                autoClosing={autoClosingPanel}
+                daoAddress={daoAddress}
+                daoStatus={daoStatus}
+                onMenuPanelClose={handleCloseMenuPanel}
+                onOpenApp={handleOpenApp}
+                opened={menuPanelOpen}
+                css="z-index: 3"
+              />
               <div
                 css={`
+                  flex-grow: 1;
+                  display: flex;
                   position: relative;
                   z-index: 1;
-                  flex-grow: 1;
-                  overflow: hidden;
                 `}
               >
-                {children}
+                <Transition
+                  config={springs.lazy}
+                  items={showAppOverlay}
+                  immediate={showAppOverlay}
+                  enter={{ opacity: 1 }}
+                  leave={{ opacity: 0 }}
+                >
+                  {show =>
+                    show &&
+                    (({ opacity }) => (
+                      <AnimatedDiv
+                        style={{ opacity }}
+                        css={`
+                          position: absolute;
+                          z-index: 2;
+                          top: 0;
+                          left: 0;
+                          right: 0;
+                          bottom: 0;
+                          background: ${theme.background};
+                        `}
+                      />
+                    ))
+                  }
+                </Transition>
+                <div
+                  css={`
+                    position: relative;
+                    z-index: 1;
+                    flex-grow: 1;
+                    overflow: hidden;
+                  `}
+                >
+                  <OrgViewApp
+                    apps={apps}
```

Comment:
```suggestion
                    activeInstanceId={locator.instanceId}
                    apps={apps}
```

The menu item highlights weren't showing, I think we need to pass `activeInstanceId` here